src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double length;
+    private double width;
+
+    public double getLength() {
+        return length;
+    }
+
+    public void setLength(double length) {
+        this.length = length;
+    }
+
+    public double getWidth() {
+        return width;
+    }
+
+    public void setWidth(double width) {
+        this.width = width;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: Rectangle, area: "
+                + getArea() + "sq. units, length "
+                + length + " units, width " + width
+                + " units, color: " + getColor().name());
+    }
+
+    public double getArea() {
+        return (length * width);
```

Comment:
Creating a figure, don't pass expressions in the constructor. Instead, create separate variables and pass them on for better code readability. It's not directly applicable here, but keep this in mind when you call the constructor of this class from elsewhere in your code.